README.md

```diff
@@ -35,20 +35,30 @@ Slack: LabPlus-team #family-searchinator
 
 ## Development
 
-to install dependencies, cd into folder and run
+Make a copy of the example environment file containing some important settings
+```
+cp example.env .env
+```
+
+To install dependencies, cd into folder and run
 ```
 npm install
 ```
 
-to run a development server:
+To run a development server:
 ```
 npm start
 ```
 
 ### Major Dependencies
+- React (^16.6.3)
+
 
 ### Quality assurance tools
 
+- jest
+- cucumber
```

Comment:
should we run `npm test` ?

Reply:
`npm test` runs the Jest tests, if that's what you're asking. The command is down in the ##Testing section. Cucumber tests aren't included at the moment because I need to look at how to configure them to run in CI (might need to `nmp start` so cucumber has something to make requests against)